source/infra/review-rdo-infra-core.html.md

```diff
@@ -0,0 +1,46 @@
+---
+title: review.rdoproject.org Infrastructure core administrators
+category: documentation,infrastructure
+---
+
+# review.rdoproject.org infrastructure core administrators
+
+The `config-core` group for review.rdoproject.org takes care of managing all configuration for the Software Factory instance used by the RDO project.
+
+## Current core list
+
+* Alan Pevec
+* Attila Darazs
+* David Moreau Simard
+* Fabien Boucher
+* Frederic Lepied
+* Javier Peña
+* Tristan Cacqueray
+
+## Review rules for changes to the config project
+
+The Software Factory config project contains the configuration for the following aspects:
+
+* Gerrit configuration (replication to GitHub)
+* Nodepool configuration (connections to clouds, quotas, etc.)
+* Zuul configuration (jobs, pipelines, 3rd party CI configuration)
+* Jenkins configuration (job definition)
+* Resource configuration (project creation, role assignments to projects)
+
+Each of the cores will probably have knowledge on some of the areas, but not on others. We are all trusted to review and approve changes on areas where we have knowledge, and seek for help of others when dealing with areas where we are not comfortable enough.
+
+The policy to merge a change is:
+
+* Reviews can be merged with a single +2 from one of the cores, if he/she is confident enough about the change (for example, the change looks ok and has been proposed by another core).
+* If the reviewer thinks that a second reviewer is required (for large/complex changes), he/she will add the other cores to the review and ask for help.
+
+## Becoming a config core
+
+The process to join the config-core group is:
+
+* Contribute to the repo by submitting changes and performing reviews on your area of expertise.
```

Comment:
I'd add a sentence to give expectations. Something like: "During that period, you must show that you got a good grasp of the platform and dedication to help maintaining it".
good grasp is the threshold to have someone to be considered as a potential core, dedication is a criteria as we want to limit the number of cores, so we will focus on people planning to contribute long-term. Why setting expectations? Simply to prevent candidates to be disappointed. Submitting changes is implied to be somehow the normal "path" to become a core but you can't prevent people thinking that "look, I submitted a lot of changes, make me core!".

Reply:
Makes sense, I've added your sentence.